lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -173,11 +173,44 @@ static LogicalResult createPoolingOp(
   Value windowTensor = rewriter.create<tensor::EmptyOp>(
       loc, getAsOpFoldResult(shape), elementType);
 
-  result = rewriter
-               .create<OpTy>(loc, outTensorInitialized.getType(),
-                             ValueRange{paddedInput, windowTensor},
-                             outTensorInitialized, stridesAttr, dilationAttr)
-               .getResult(0);
+  SmallVector<int64_t> dimensions;
+  Value permutedInput = paddedInput, permutedOutput = outTensorInitialized;
+  if (dimensionality == 3) {
+    // Permute input and output tensor as follows:
+    // (n,c,d,h,w) -> (n,d,h,w,c)
+    dimensions = {0, 2, 3, 4, 1};
```

Comment:
Declare these in here. The values do not appeared to be used elsewhere so now point to have it declared outside of the if condition.